sschema/handler/include.py

```diff
@@ -16,22 +16,18 @@
 
 def _handler(include_paths, uri):
     '''Handles an URI of form include://'''
-    base = 'include://'
-    assert(uri.startswith(base))
-    uri = uri[len(base):]
+    scheme = 'include://'
+    assert(uri.startswith(scheme))
+    uri = uri[len(scheme):]
 
-    data = None
-    for path in include_paths:
-        uri = os.path.join(path, uri)
+    for base in include_paths:
+        path = os.path.join(base, uri)
         try:
-            f = open(uri, 'r')
+            with open(path, 'r') as f:
+                return yaml.safe_load(f)
         except FileNotFoundError:
             continue
-        else:
```

Comment:
Sorry, small nit: we should keep the `else` statement. Its purpose is so the `FileNotFoundError` wrapping covers the fewest lines possible... it should not cover `yaml.load(f)`.